control_plane/src/background_process.rs

```diff
@@ -179,11 +195,6 @@ pub fn stop_process(immediate: bool, process_name: &str, pid_file: &Path) -> any
         match process_has_stopped(pid) {
             Ok(true) => {
                 println!("\n{process_name} stopped");
-                if let Err(e) = fs::remove_file(pid_file) {
```

Comment:
With this, nobody is removing the pidfile anymore, right?

Reply:
Correct